internal/nodes/avi_model_l4_translator.go

```diff
@@ -60,7 +60,13 @@ func (o *AviObjectGraph) ConstructAviL4VsNode(svcObj *corev1.Service, key string
 			fqdns = append(fqdns, fqdn)
 		}
 	}
+	// Default: Traffic is enabled
+	// Not using it in Shared VIP as we don't have use case for it.
+	trafficEnabled := proto.Bool(true)
 
+	if traffic_enabled, ok := svcObj.Annotations[lib.VSTrafficEnabled]; ok && strings.ToLower(traffic_enabled) == "false" {
```

Comment:
This logic should also go to avi_model_advl4_translator.go file

Reply:
I think there's a little confusion here, we need this logic to be present in advl4_translator as well for this to work in VCF deployments.